zproject/computed_settings.py

```diff
@@ -1209,16 +1208,6 @@ def file_handler(
         path = f"/etc/zulip/saml/idps/{idp_name}.crt"
     idp_dict["x509cert"] = get_from_file_if_exists(path)
 
-    if "zulip_groups" in idp_dict.get("extra_attrs", []):
-        raise AssertionError("zulip_groups can't be listed in extra_attrs in the IdP config.")
-
-
-for dict_for_subdomain in SOCIAL_AUTH_SYNC_ATTRS_DICT.values():
-    for attrs_map in dict_for_subdomain.values():
-        if "zulip_groups" in attrs_map.values():
```

Comment:
Was this actually meant to check `.keys()`?

Reply:
That does seem like what was intended!